src/style.css

```
@@ -1 +1,30 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 div:nth-child(n + 1),
+.stars--1 div:nth-child(n + 2),
+.stars--2 div:nth-child(n + 3),
+.stars--3 div:nth-child(n + 4),
+.stars--4 div:nth-child(n + 5) {
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

```suggestion
.stars--0 div:nth-child(n + 1),
.stars--1 div:nth-child(n + 2),
.stars--2 div:nth-child(n + 3),
.stars--3 div:nth-child(n + 4),
.stars--4 div:nth-child(n + 5) {
  background-image: url('./images/star.svg');
}
```
you specified it in the .stars__star